css/style.css

```diff
@@ -0,0 +1,448 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.navbar {
+  width: 100%;
+  height: auto;
+  padding: 10px 10px 20px 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.menu-icon {
+  padding: 30px 10px 10px 20px;
+  width: 43px;
+  height: auto;
+}
+
+.m-menu {
+  width: 30px;
+  margin-bottom: 3px;
+  margin-bottom: 10%;
+  border: 3px solid #272a31;
+}
+
+.head-line {
+  display: none;
+  width: 100%;
+  height: 50px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: #3e3e3e;
+  flex-direction: row;
+  justify-content: flex-end;
+  padding: 20px 20px;
+}
+
+.head-img {
+  width: 25px;
+  height: auto;
+  display: flex;
+  align-self: center;
+  margin-right: 20px;
+}
+
+.head-link {
+  color: #fff;
+  margin-right: 20px;
+}
+
+.menu-ul {
+  list-style: none;
+  display: none;
+  flex-direction: column;
+  width: 100%;
+  height: auto;
+  padding-left: 2%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: #fff;
+  margin-bottom: 20%;
+}
+
+.close {
+  width: 50px;
+  height: 50px;
+  margin-top: 20px;
+}
+
+.header-logo {
+  display: none;
+}
+
+.links {
+  padding-top: 5px;
+  margin: 0 0 10px 10px;
+}
+
+.menu-links {
+  text-decoration: none;
+  color: #272a31;
+}
+
+.menu-ul.active {
+  display: flex;
+}
+
+.pg1-sec {
+  background-image: url("../Images/background.png");
+  height: auto;
+  padding: 80px 50px;
+  width: 100%;
+  padding-top: 210px;
+}
+
+@font-face {
+  font-family: cocogoose;
+  src: url(../Images/fonts/Cocogoose.ttf);
+}
+
+.hello-world {
+  font-size: 25px;
+  font-family: 'Lato', sans-serif;
+  color: #ec5242;
+  margin-bottom: 10px;
+}
+
+.event-name {
+  font-size: 30px;
+  font-family: 'cocogoose', sans-serif;
+  opacity: 90%;
```

Comment:
[OPTIONAL]

This title seems to be overlapping the navigation bar because of its `opacity` property :mag:
![image](https://user-images.githubusercontent.com/78335447/175751469-de908ff6-a193-4b9e-86f0-ab990da8df85.png)

Consider utilizing [a tool](http://marcodiiga.github.io/rgba-to-rgb-conversion) to get the color right without having to change the opacity, or to use `rgba` colors instead of `hex` :100: